### DISCUSSION TITLE
Filter for No Collection

### DISCUSSION THREAD

**Discussion by Noggog** [Kareadita/Kavita#3466] [Ideas]
### Idea Description

When adding new content I like adding everything to a collection, as the raw library can be hard to navigate sometimes.
However, it is hard for even me to pick out the new content I've added to add it to the new collection.

It would be nice to have a filter that would only show items that are not included in any collection.

I tried this: 
![image](https://github.com/user-attachments/assets/2e34d0f9-c272-400f-8732-0599abfbedf2)

But that did not have the intended effect

### Idea Category

Feature Enhancement

### Duration of Using Kavita

Several Months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by DieselTech** [comment]
What effect did the filter you made have? That way we can make sure it is working as intended.

**Reply by Noggog** [reply]
It behaves as if the `Collection Tags` entry did not exist at all, and shows everything.

**Comment by SqueezedByte** [comment]
Also I wish for this to be implemented. There is the filter on collection tag but it don't work.

**Comment by majora2007** [comment]
I just took a look and this should be a bug, not a Feature request. I'll convert it for you

**Reply by majora2007** [reply]
https://github.com/Kareadita/Kavita/issues/3717

**Reply by majora2007** [reply]
This will come with v0.8.6 release. For those on this, next time please make sure you create an issue rather than a feature request for when things are broken. This could have been fixed a lot earlier if it was appropriately created. Thanks.
